### PR TITLE
Create core values section for the html

### PR DESCRIPTION
**Pull request by NazgulM**
- Adding content to index.html
- Adding some content from Bootstrap
- Using CSS

### CODE REVIEW

File: styles/something.css [hunk 1]
```diff
@@ -2,6 +2,118 @@ body {
...
+.col {
+  display: flex;
+  flex-wrap: wrap;
+  -webkit-box-flex: 1;
+  flex-grow: 1;
+  -webkit-box-pack: justify;
+  justify-content: space-between;
+}
+
+img {
```
Comment (alihaidermalik20) [inline styles/something.css:55]: this makes all the images that will be in our project 50px

Comment (NazgulM) [reply styles/something.css:55]: this img not mine, I used .card-img-top

Comment (NazgulM) [reply styles/something.css:55]: I can edit Maha's img

Comment (NazgulM) [reply styles/something.css]: Done

**Review by Shewitttt — APPROVED** [review]

**Pull request merged**